scripts/manage-versions.ts

```diff
@@ -61,8 +65,8 @@ function saveFiles(version, data) {
   const dataFile = `${schemaPath}/${version}.json`;
   const treeFile = `${schemaPath}/${version}_tree.json`;
   fse.writeJSONSync(dataFile, data);
-  const treeData = generateTreeData(data);
-  fse.writeJSONSync(treeFile, treeData);
+  // const treeData = generateTreeData(data);
+  // fse.writeJSONSync(treeFile, treeData);
```

Comment:
worth keeping around or should we just remove?

Reply:
worth keeping for now, i think tree will be worked on pretty soon